build/build.xml

```diff
@@ -241,6 +241,79 @@
 		</sequential>
 	</macrodef>
 
+	<macrodef name="build-addon-without-help-indexes" description="build the specified addon">
```

Comment:
Ok, thanks. I'll leave it as-is then. Can we merge it and can I do the same on the other branches?